lib/omniauth/strategies/latvija/signed_document.rb

```diff
@@ -15,79 +15,92 @@
 # If applicable, add the following below the CDDL Header,
 # with the fields enclosed by brackets [] replaced by
 # your own identifying information:
-# "Portions Copyrighted [year] [name of copyright owner]"
+# 'Portions Copyrighted [year] [name of copyright owner]'
 #
 # $Id: xml_sec.rb,v 1.6 2007/10/24 00:28:41 todddd Exp $
 #
 # Copyright 2007 Sun Microsystems Inc. All Rights Reserved
 # Portions Copyrighted 2007 Todd W Saxton.
 
-module OmniAuth
-  module Strategies
-    class Latvija
-      class SignedDocument < REXML::Document
-        DSIG = "http://www.w3.org/2000/09/xmldsig#"
+module OmniAuth::Strategies
+  class Latvija
+    class SignedDocument
+      DSIG = 'http://www.w3.org/2000/09/xmldsig#'.freeze
+      XENC = 'http://www.w3.org/2001/04/xmlenc#'.freeze
+      CANON_CLASS = Nokogiri::XML::XML_C14N_EXCLUSIVE_1_0
```

Comment:
`CANON_MODE` would be a better name, as `mode` is used as the parameter name in nokogiri's `canonicalize ` method.